src/allotropy/parsers/qiacuity_dpcr/qiacuity_dpcr_parser.py

```diff
@@ -0,0 +1,248 @@
+            )
+        )
+
+    def _get_dpcr_document(self, qiacuity_data: pd.DataFrame):
+        measurement_documents = []
+        for _, well_item in qiacuity_data.iterrows():
+            sample_document = self._get_sample_document(well_item=well_item)
+            device_control_documents = [self._get_device_control_document()]
+            device_control_aggregate_document = DeviceControlAggregateDocument(
+                device_control_document=device_control_documents
+            )
+            processed_data_aggregate_document = ProcessedDataAggregateDocument(
+                processed_data_document=[
+                    self._get_processed_data_document(
+                        well_item=well_item,
+                        dp_document=self._get_data_processing_document(
+                            well_item=well_item
+                        ),
+                    )
+                ]
+            )
+            measurement_document = self._get_measurement_document(
+                well_item=well_item,
+                sample_document=sample_document,
+                device_control_aggregate_document=device_control_aggregate_document,
+                processed_data_aggregate_document=processed_data_aggregate_document,
+            )
+            measurement_documents.append(measurement_document)
+        # TODO: Hardcoded plate well count to 0 since it's a required field
+        #  ASM will be modified to optional in future version
+        measurement_aggregate_document = MeasurementAggregateDocument(
+            measurement_document=measurement_documents,
+            plate_well_count=TQuantityValueNumber(value=0),
+        )
+        return DPCRDocumentItem(
+            measurement_aggregate_document=measurement_aggregate_document
+        )
+
+    def _get_device_system_document(self) -> DeviceSystemDocument:
+        device_system_document = DeviceSystemDocument(
+            device_identifier=DEVICE_IDENTIFIER,
+            brand_name=BRAND_NAME,
+            product_manufacturer=PRODUCT_MANUFACTURER,
+        )
+        return device_system_document
+
+    def _get_data_system_document(self, file_name: str) -> DataSystemDocument:
+        data_system_document = DataSystemDocument(
+            file_name=file_name,
+            software_name=SOFTWARE_NAME,
+            ASM_converter_name=ASM_CONVERTER_NAME,
+            ASM_converter_version=ASM_CONVERTER_VERSION,
+        )
+        return data_system_document
+
+    def _get_measurement_document(
+        self,
+        well_item: pd.Series,
+        sample_document: SampleDocument,
+        device_control_aggregate_document: DeviceControlAggregateDocument,
+        processed_data_aggregate_document: ProcessedDataAggregateDocument,
+    ) -> MeasurementDocumentItem:
+        # Assign a random measurement id
+        measurement_id = random_uuid_str()
+        # There is no measurement time in the file, so assign to unix epoch
+        measurement_time = EPOCH
```

Comment:
I wonder if we could use file creation time? I know you wrote the requirements on this, so I'll defer to you if you think this is a reasonable interpretation.

Reply:
Yeah I talked to Joe about this today, sounds like he's ok leaving it for now